guides/doc-Managing_Hosts/topics/ref_customizing-the-global-registration-and-host-registration-templates.adoc

```diff
@@ -0,0 +1,94 @@
+
+.Templates
+
+The registration process uses the following registration templates:
+
+* The *Global Registration* template contains steps for registering hosts to {Project}. This template renders when hosts access `/register` endpoint.
+
+* The *Linux registration default* template contains steps for configuring hosts after they are registered.
+
+.Global Parameters
+
+You can configure the following global parameters by navigating to *Configure* > *Global Parameters*:
+
+* The `host_registration_remote_execution` parameter is used in the `remote_execution_ssh_keys` snippet, the default value is `true`.
+
+ifeval::["{build}" != "satellite"]
+* The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `false`.
+endif::[]
+
+ifeval::["{build}" == "satellite"]
+* The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `true`.
+endif::[]
+
+.Snippets
+
+`remote_execution_ssh_keys` snippet is used in the *Linux registration default* template.
+It deploys SSH keys to the host only when the `host_registration_remote_execution` parameter is `true`.
+
+`insights` snippet is used in the *Linux registration default* template.
+It downloads and installs the Red{nbsp}Hat Insights client when global parameter `host_registration_insights` is set to true.
+
+.Variables
+
+This table describes what variables are used in the `Global Registration` template.
+
+.The Global Registration Template Variables
+[cols=3*,options=header]
+|===
+|Variable
+|Command argument
+|Description
+
+|`@user`
+|none
+|Current authenticated user object.
+
+|`@organization`
+|`organization_id`
+|If `organization_id` is not set, then user's default organization is set, or the first organization from user's organizations list.
+
+|`@location`
+|`location_id`
+|If `location_id` is not set, user's default location is set, or the first location from user's locations list.
+
+|`@hostgroup`
+|`hostgroup_id`
+|Host group of the host.
+
+|`@operatingsystem`
+|`operatingsystem_id`
+|Host operating system.
+
+|`@setup_insights`
+|`setup_insights`
+|Override the value of the `host_registration_insights` global parameter for the registered host and install insights client.
+
+|`@setup_remote_execution`
+|`setup_remote_execution`
+|Override the value of `host_registration_remote_execution` global parameter for the registered host and deploy SSH keys for remote execution.
+
+|`@remote_execution_interface`
+|`remote_execution_interface`
+|Set default interface of host for the remote execution.
+
+|`@activation_key`
+|`activation_key`
+ifeval::["{build}" == "satellite"]
+|Host activation keys.
+endif::[]
+ifeval::["{build}" != "satellite"]
+|This snippet is for the Katello users only. Host activation keys.
```

Comment:
```suggestion
|The Host activation keys snippet is for Katello users only.
```
or just:
This snippet is for Katello users only.
No need for "the" as you have not identified the katello users.

Reply:
Done.